sunbeam-python/sunbeam/versions.py

```diff
@@ -71,8 +77,81 @@
 K8S_SERVICES |= OPENSTACK_SERVICES_K8S
 K8S_SERVICES |= OVN_SERVICES_K8S
 K8S_SERVICES |= MYSQL_SERVICES_K8S
+K8S_SERVICES |= MYSQL_ROUTER_SERVICES_K8S
 K8S_SERVICES |= MISC_SERVICES_K8S
 
 CHARM_VERSIONS = {}
 CHARM_VERSIONS |= K8S_SERVICES
 CHARM_VERSIONS |= MACHINE_SERVICES
+
+snap = Snap()
+TERRAFORM_PLANS = {
+    "sunbeam-machine-plan": {
+        "source": Path(snap.paths.snap / "etc" / "deploy-sunbeam-machine")
+    },
+    "microk8s-plan": {"source": Path(snap.paths.snap / "etc" / "deploy-microk8s")},
+    "microceph-plan": {"source": Path(snap.paths.snap / "etc" / "deploy-microceph")},
+    "openstack-plan": {"source": Path(snap.paths.snap / "etc" / "deploy-openstack")},
+    "hypervisor-plan": {
+        "source": Path(snap.paths.snap / "etc" / "deploy-openstack-hypervisor")
+    },
+    "demo-setup": {"source": Path(snap.paths.snap / "etc" / "demo-setup")},
+}
+
+K8S_CHARMS = {}
+K8S_CHARMS |= OPENSTACK_SERVICES_K8S
+K8S_CHARMS |= OVN_SERVICES_K8S
+K8S_CHARMS |= MYSQL_SERVICES_K8S
+K8S_CHARMS |= MISC_SERVICES_K8S
+DEPLOY_OPENSTACK_TFVAR_MAP = {
+    svc: {
+        "channel": f"{svc}-channel",
+        "revision": f"{svc}-revision",
+        "config": f"{svc}-config",
+    }
+    for svc, channel in K8S_CHARMS.items()
+}
+DEPLOY_OPENSTACK_TFVAR_MAP.pop("traefik-public")
+DEPLOY_OPENSTACK_TFVAR_MAP["keystone-mysql-router"] = {
+    "channel": "mysql-router-channel",
+    "revision": "mysql-router-revision",
+    "config": "mysql-router-config",
+}
```

Comment:
Isn't that already populated by the by L112 ?

Reply:
The idea is to use mysql-router in manifest instead of *-mysql-router.
Changed little bit to reflect the same instead of keystone-mysql-router.